# An array of phase space markers with a header for global reference

std_msgs/Header header
phase_space/PhaseSpaceMarker[] markers
